Tasks/AzureRmWebAppDeployment/azurermwebappdeployment.ts

```diff
@@ -36,6 +36,11 @@ async function run() {
         SPN["tenantID"] = endPointAuthCreds.parameters["tenantid"];
         SPN["subscriptionId"] = tl.getEndpointDataParameter(connectedServiceName, 'subscriptionid', true);
 
+        var availableWebPackages = tl.glob(webDeployPkg);
```

Comment:
Cover the following scenarios

1. No package matching the specified pattern --> It should fail
2. Only one package matching specified pattern --> It should pass
3. More than one package found --> It should fail